### PR TITLE
[SDPA-3202] Share Links - Test Environment. DO NOT MERGE.

### PR DESCRIPTION
**Pull request by alan-cole**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-3202

### Changed

1.  This PR is https://github.com/dpc-sdp/ripple/pull/786 but with the .env set to https://nginx-php-content-vic-pr-971.lagoon.vicsdp.amazee.io/

### CODE REVIEW

File: packages/ripple-nuxt-tide/lib/core/tide.js [hunk 1]
```diff
@@ -443,6 +448,33 @@ export const tide = (axios, site, config) => ({
     return pageData
   },
 
+  getPageByShareLink: async function (path, section) {
```
Comment (tim-yao) [inline packages/ripple-nuxt-tide/lib/core/tide.js:451]: @alan-cole Can we split this function to different file? I would like to make tide.js only focus on the basic Tide API communication. Thanks!

**Review by tim-yao — COMMENTED** [review]
Hey @alan-cole I think if we turn off authenticated content module, this should still work. Right?

File: packages/ripple-nuxt-tide/lib/core/tide.js (hunk 1 above)
Comment (alan-cole) [reply packages/ripple-nuxt-tide/lib/core/tide.js:451]: I'll create a submodule for sharelinks and add this in.

**Comment by alan-cole** [comment]
> Hey @alan-cole I think if we turn off authenticated content module, this should still work. Right?

Hi Tim, that's correct. This should still be working without authenticated content.

**Comment by alan-cole** [comment]
Closing as Share links is now part of develop.

**Pull request closed without merging**